src/modules/content-files/filedb.h

```diff
@@ -0,0 +1,43 @@
+/************************************************************\
+ * Copyright 2020 Lawrence Livermore National Security, LLC
+ * (c.f. AUTHORS, NOTICE.LLNS, COPYING)
+ *
+ * This file is part of the Flux resource manager framework.
+ * For details, see https://github.com/flux-framework.
+ *
+ * SPDX-License-Identifier: LGPL-3.0
+\************************************************************/
+
+#ifndef _CONTENT_FILES_FILEDB_H
+#define _CONTENT_FILES_FILEDB_H
+
+/* Read file named 'key' from the dbpath directory.
+ * On success, 'datap' and 'sizep' are assigned the contents and size
+ * and 0 is returned.  On failure, -1 is returned with errno set.
+ * Pass '*errstr' in pre-set to NULL and if a human readable error message
+ * is appropriate, it is assigned on error.
+ */
```

Comment:
should clarify `datap` should be freed, but `errstr` not freed?

Reply:
Will do!